src/boardlib/__main__.py

```diff
@@ -30,10 +30,12 @@ def logbook_entries(board, username, password, grade_type="font", database=None)
 
 
 def write_entries(output_file, entries, no_headers=False, fields=LOGBOOK_FIELDS):
+    cleaned_entries = [{k: v for k, v in entry.items() if k in fields} for entry in entries]
```

Comment:
This will be more efficient as a generator instead of materializing it into an intermediate list, as `writerows` accepts any iterable.

```suggestion
    cleaned_entries = ({k: v for k, v in entry.items() if k in fields} for entry in entries)
```